x-pack/platform/plugins/shared/actions/server/types.ts

```diff
@@ -118,11 +118,9 @@ export type ExecutorType<
   options: ActionTypeExecutorOptions<Config, Secrets, Params>
 ) => Promise<ActionTypeExecutorResult<ResultData>>;
 
+type Validator<T> = Pick<z.ZodType<T>, 'parse'>;
 export interface ValidatorType<T> {
-  schema: {
-    validate(value: unknown): T;
-    getSchema?: () => AnySchema;
-  };
+  schema: Validator<T>;
```

Comment:
Is `parse()` enough here?  Or I guess if we need other schema accessors we could just add them?